docs/INSTALL.md

```diff
@@ -43,12 +43,21 @@ Configure the repository for an opensource build:
 bash ./configure.sh
 ```
 
-For _Windows 10_: Use WSL to run `./configure.sh`, or, alternatively, run the following command from the
-[Visual Studio Developer Command Prompt](https://docs.microsoft.com/en-us/visualstudio/ide/reference/command-prompt-powershell?view=vs-2022)
-and follow instructions:
+For _Windows 10_:  You should be able to build the project by following either of these setups:
+
+**Setup 1: Using WSL**
+1. Install [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) on your machine.
+2. Install g++ by running the following command in WSL: `sudo apt install g++`
+3. Execute `bash ./configure` in WSL.
+4. After bootstrapping, don't forget to generate header files by `cd 3party/boost` and executing `./b2 headers`.
```

Comment:
This step should not be needed, right? Can you please double-check it, by running
```
git clean -xfd
git submodule foreach --recursive git clean -xfd
```
and then repeating these steps?

Reply:
Ok, now I know step 4 isn't required.